gdc_client/utils.py

```diff
@@ -0,0 +1,14 @@
+from urllib import urlencode
```

Comment:
will break in python3
```
if sys.version_info >= (3,):
    import urllib.parse as urlparse
    from urllib.parse import urlencode
else:
    import urlparse
    from urllib import urlencode

```

Reply:
this isn't the only thing that's going to break with python3 😄